cylc/flow/task_events_mgr.py

```diff
@@ -798,7 +798,9 @@ def _process_message_submitted(self, itask, event_time):
                 '[%s] -job[%02d] submitted to %s:%s[%s]',
                 itask,
                 itask.summary['submit_num'],
-                itask.summary['host'],
+                itask.summary['platforms_used'][max(
+                    itask.summary['platforms_used'].keys()
+                )],
```

Comment:
Is the max getting the submit num? If so can just use `itask.summary['submit_num']`?